functions.php

```diff
@@ -31,23 +31,28 @@ function displayChairs($query): string
 }
 
 /** create a function to insert the POST form data into the database
+ * @param PDO $db
 
-function insertChair()
-{
-    if(isset($_POST['submit']))
+ */
+
+function insertChair(PDO $db){
+
+    if(isset($_POST['submit'])
+        && (!empty($_POST['chairname']) && !empty($_POST['designer']) && !empty($_POST['designyear']) && !empty($_POST['imagelink'])))
```

Comment:
I think mike said if there are multiple conditionals in an if statement to wack them in a function?

Reply:
but maybe ask ash ?

Reply:
Will do.